lib/std/private/strimpl.nim

```diff
@@ -0,0 +1,41 @@
+func toLowerAscii*(c: char): char {.inline.} =
```

Comment:
As I said before, the benchmark in that PR is wrong, `chr(ord(c) + (ord('a') - ord('A')))` may be not slower than `char(uint8(c) xor 0b0010_0000'u8)`

Reply:
Especially we does have a better optimization as did in `compiler/strutils2.nim`
`result = chr(c.ord + (if c.isUpperAscii: (ord('a') - ord('A')) else: 0))`

```
  # refs https://github.com/timotheecour/Nim/pull/54
  # this is 10X faster than a naive implementation using a an optimization trick
  # that can be adapted in similar contexts. Predictable writes avoid write
  # hazards and lead to better machine code, compared to random writes arising
  # from: `if c.isUpperAscii: c = ...`
```

Reply:
`{.noinit.}` ?.